src/math/FGFunction.h

```diff
@@ -735,7 +735,7 @@ class FGFunction : public FGParameter, public FGJSBBase
   /// Default constructor.
   FGFunction()
     : cached(false), cachedValue(-HUGE_VAL), pNode(nullptr), pCopyTo(nullptr),
-      PropertyManager(nullptr) {}
+      PropertyManager(nullptr), constForConstParams(true) {}
```

Comment:
Remove the `constForCostParams` member, it is not needed (see comment below in `FGFunction::IsConstant`)